jrpc/jrpc_test.go

```diff
@@ -270,7 +270,7 @@ func TestJPRC(t *testing.T) {
 	if unwrap.Error() != expected {
 		t.Errorf("unwrapped error expected to be %q, got %v", expected, unwrap.Error())
 	}
-	expected = "deserialization error, code 747: " + expected + " (raw reply: {bad})"
+	expected = "non ok http result and deserialization error, code 747: " + expected + " (raw reply: {bad})"
```

Comment:
```suggestion
	expected = "non ok http result and deserialization error, code 747: " + expected + " - raw reply: {bad}"
```
Removing the `()` - personal nit

Reply:
that's the test :) - and in general the reason it's in () is the raw reply isn't typically necessary to be looked at